openmls/src/framing/validation.rs

```diff
@@ -81,7 +81,7 @@ impl DecryptedMessage {
             let ciphersuite = group.ciphersuite();
             let (message_secrets, old_leaves) = group
                 .message_secrets_and_leaves_mut(ciphertext.epoch())
-                .map_err(|_| MlsCiphertextError::DecryptionError)?;
+                .map_err(|_| MessageDecryptionError::DecryptionError)?;
```

Comment:
`MessageDecryptionError::DecryptionError` seems a little odd. I suppose all errors in that enum are "DecryptionErrors".